package.json

```diff
@@ -9,29 +9,32 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@headlessui/react": "^1.7.16",
+    "@heroicons/react": "^2.0.18",
     "@types/node": "20.3.3",
     "@types/react": "18.2.14",
     "@types/react-dom": "18.2.6",
     "autoprefixer": "10.4.14",
     "eslint-config-next": "13.4.7",
     "eslint-config-prettier": "^8.8.0",
     "eslint-plugin-prettier": "^4.2.1",
+    "i": "^0.3.7",
     "i18next": "^23.2.11",
     "next": "13.4.7",
+    "next-auth": "^4.22.5",
```

Comment:
❌ 
if you are not implementing the next-auth in this PR, then please remove this

Reply:
Done